src/components/layout/nav-layouts/RegisterNavLinks.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
```

Comment:
Added simple Nav component for the register page to show single Log in, but have it associated as a NavBar.